[doc] Update hello_world.md (#6889)
taichi-dev/taichi: docs/lang/articles/get-started/hello_world.md

</center>

:::note

If you are *not* using an IDE for running code, you can simply run the code from your terminal by:

1. Go to the directory that contains your .py file
2. Type in `python3 *nameOfFile*.py` (replace *nameOfFile* with your programs name. Be sure to include the `.py` extension)

:::

Let's dive into this simple Taichi program.

### Import Taichi

The first two lines import Taichi and its `math` module. The `math` module contains built-in vectors and matrices of small dimensions, such as `vec2` for 2D real vectors and `mat3` for 3&times;3 real matrices.

See the [Math Module](../math/math_module.md) for more information.

### Initialize Taichi

:::note

> `ti.init(x:Optional)`- Initializes Taichi environment and allows you to customize your Taichi runtime depending on the `x` number of arguements passed into it.

Reviewer: ```suggestion
`ti.init(**kwargs)`- Initializes Taichi environment and allows you to customize your Taichi runtime depending on the optional arguments passed into it.
```

:::

For now, we only introduce the most important argument, namely, `arch`.

The argument `arch` specifies the *backend* that executes the compiled code. A backend can be either `ti.cpu` or `ti.gpu`. When `ti.gpu` is specified, Taichi moves down the backend list of `ti.cuda`, `ti.vulkan`, and `ti.opengl/ti.metal`. If no GPU architecture is available, Taichi falls back to your CPU device.

You can also directly specify which GPU backend to use. For example, set `arch=ti.cuda` to run your program on CUDA. Taichi raises an error if the target architecture is unavailable. See the [Global Settings](../reference/global_settings.md) for more information about `ti.init()`.

> ```python{4}
> import taichi as ti
> import taichi.math as tm
> 
> ti.init(arch=ti.gpu)
> ```

Reviewer: ```suggestion
```python{4}
import taichi as ti
import taichi.math as tm

ti.init(arch=ti.gpu)
```
A question here: is it intended to move all three lines at the end of the section `Initialize Taichi`? 
```

Author: Yes, in the first meeting we had we touched upon having the dialogue before the code snippet. If you'd like, I can add some extra dialogue to explain the code itself.

> ### Define a Field

Reviewer 2: ```suggestion
### Define a Taichi field
```

:::note

> `ti.field(x,y)` - Defines a Taichi field whose shape is of `y` and whose elements are of type `x`.

Reviewer: ```suggestion
`ti.field(dtype, shape)` - Defines a Taichi field whose shape is of `shape` and whose elements are of type `dtype`.
```

*Field* is the most important and frequently used data structure in Taichi. You can compare it to NumPy's `ndarray` or PyTorch's `tensor`. But Taichi's Field is more flexible. For example, a Taichi Field can be [spatially sparse](../basic/sparse.md) and easily [switched between different data layouts](../basic/layout.md).

> We will introduce more advanced features of field in other scenario-based tutorials. For now, it suffices to know that the Field `pixels` is a dense 2D array.

Reviewer 2: ```suggestion
We will introduce more advanced features of fields in other scenario-based tutorials. For now, it suffices to know that the field `pixels` is a dense 2D array.
```
